### DISCUSSION TITLE
Code pin au lieu de mot de passe

### DISCUSSION THREAD

**Discussion by uurshin** [fibreville/minotaure_php#43] [Boite à idées]
Il faudrait empêcher les joueurs de rentrer leur mot de passe habituel puisqu'on ne peut pas garantir que les gens installent l'appli sur un serveur sécurisé. Alors certes c'est leur propre responsabilité, mais vu que j'ai fait une vidéo pour installer sur hébergeur gratos, sans SSL, on les incite à aller dessus. 

Je pensais passer par un système de code pin à 4 chiffres. Après tout pas besoin d'un code compliqué pour un système aussi peu critique. Qu'en pensez-vous ?

**Comment by ElethornGM** [comment]
C'est tellement une bonne idée!

**Comment by asaurat** [comment]
Ce serait top en effet !
